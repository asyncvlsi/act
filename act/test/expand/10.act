

template <pint i>
defproc foo (bool x[i]; bool y[i])
{
  prs {
   (k:i: x[k] => y[k]-)
  }
  foo<i-1> f;
}


foo<3> x;

foo<2+1> y;
