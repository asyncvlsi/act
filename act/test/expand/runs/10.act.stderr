In expanding foo<0>
In expanding foo<1>
In expanding foo<2>
In expanding foo<3>
In expanding ::<Global>
 id: x[0]
 type: bool[0]
FATAL: Dereference out of range
